packages/itwin/measure-tools/src/api/SheetMeasurementHelper.ts

```diff
@@ -159,16 +159,47 @@ export namespace SheetMeasurementsHelper {
    * @param allowedDrawingTypes
    * @returns
    */
-  export function checkIfAllowedDrawingType(ev: BeButtonEvent, allowedDrawingTypes: DrawingType[]) {
-    if (!ev.viewport)
+  export function checkIfAllowedDrawingType(viewport: ScreenViewport | undefined, point: Point3d, allowedDrawingTypes: DrawingType[]) {
+    if (!viewport)
       return false;
-    for (const drawing of DrawingDataCache.getInstance().getSheetDrawingDataForViewport(ev.viewport)) {
+    for (const drawing of DrawingDataCache.getInstance().getSheetDrawingDataForViewport(viewport)) {
       if (allowedDrawingTypes.includes(drawing.type)) {
-        if (SheetMeasurementsHelper.checkIfInDrawing(ev.point, drawing.origin, drawing.extents)) {
+        if (SheetMeasurementsHelper.checkIfInDrawing(point, drawing.origin, drawing.extents)) {
           return true;
         }
       }
     }
     return false;
   }
+
+  function getNameFromDrawingType(type: SheetMeasurementsHelper.DrawingType): string  {
```

Comment:
you should NOT be using strings from other namespaces
if you need these strings, copy them into this pkg

Reply:
Oops, my bad, I originally made this change in the other package and I forgot this change! It should be good now

Reply:
Arun for the save...I completely glossed over CivilReviewTools...